my-app/src/pages/closet.js

```diff
@@ -1,12 +1,113 @@
 import React, {useEffect, useState} from "react";
+import AddIcon from '@mui/icons-material/Add';
 import Grid from "@mui/material/Grid";
 import Typography from "@mui/material/Typography";
 import ClothingItem from "../component/ClothingItem";
 import Box from "@mui/material/Box";
+import IconButton from '@mui/material/IconButton';
+import BasicSelect from "../component/textField";
+import {Modal, Button, TextField, InputLabel, MenuItem, FormControl} from '@mui/material'
+import Select from 'react-select'
 
 
-const Closet = () => {
+const Closet = (props) => {
   let [clothingItems, setClothingItems] = useState([])
+  const [addItemModal, openAddItem] = useState(false);
+
+  const userId = props.userId
+
+  const handleAddItem = () =>{
+    openAddItem(true)
+  }
+  const handleCloseModal = () => {
+    openAddItem(false)
+  }
+
+  const [enteredItemName, setItemName] = useState("");
+  const [selectedType, setSelectedType] = useState("");
+  const [selectedWeatherAttributes, setWeatherAttributes] = useState([]);
+
+
+  const handleChangeWeatherAttributes= (attributes) => {
+    let items = []
+    attributes.map(item => {
+      items.push(item.value)
+    })
+    setWeatherAttributes(items)
+  };
+
+  const handleChangeItemName = (name) =>{
+    setItemName(name)
+  }
+
+  const handleChangeType= (type) => {
+    setSelectedType(type.value);
+  };
+  
+  const handleOnClick = () => {
+    selectedWeatherAttributes.push(selectedType)
+    let clothingItem = {name: enteredItemName, attributes: selectedWeatherAttributes}
+    fetch('/dummy/postOutfit', {
```

Comment:
Can we pick a different name for this? 'outfit' implies it's not just a single item.

(I also recommend removing `post` from the name because that is implied by the request type, no big deal though)